packages/mobile/src/screens/product-info/components/description/description.tsx

```diff
@@ -0,0 +1,69 @@
+import { ProductDto } from '@vse-bude/shared';
+import React, { FC } from 'react';
+import { DotSvg } from '~/assets/svg/dot';
+import { Text, View } from '~/components/components';
+import { useCustomTheme, useTranslation } from '~/hooks/hooks';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const Description: FC<Partial<ProductDto>> = ({
+  description,
+  city,
+  status,
+  endDate,
+}) => {
+  const { colors } = useCustomTheme();
+  const { t } = useTranslation();
+
+  const date = endDate
+    ?.toISOString()
+    .split('T')[0]
+    .split('-')
+    .reverse()
+    .join('.');
+  const renderInfo = (title: string, description: string) => {
+    return (
+      <View style={[globalStyles.flexDirectionRow, globalStyles.py2]}>
+        <View
+          style={[
+            styles.title,
+            globalStyles.flexDirectionRow,
+            globalStyles.alignItemsCenter,
+          ]}
+        >
+          <DotSvg />
```

Comment:
Why don't you just use icon?

Reply:
couldn't find dot icon in Feather icons, and plus icon didn't respond design and looked not well